utils/node_local_test.py

```diff
@@ -372,12 +373,16 @@ def start(self):
                                                      suffix='.log',
                                                      delete=False)
 
-        self._agent = subprocess.Popen([agent_bin,
-                                        '--config-path', agent_config,
-                                        '--insecure',
-                                        '--debug',
-                                        '--runtime_dir', self.agent_dir,
-                                        '--logfile', agent_log_file.name],
+        agent_cmd = [agent_bin,
+                     '--config-path', agent_config,
+                     '--insecure',
+                     '--runtime_dir', self.agent_dir,
+                     '--logfile', agent_log_file.name]
+
+        if not self.conf.args.server_debug:
```

Comment:
not understanding the relationship between server_debug log mask setting and whether agent should start with --debug
Is intent perhaps to be this instead (use --debug if not running a performance check)?
if not self.conf.args.perf_check:

Reply:
The intention with NLT was that it would run with *all* debugging on all the time, only run very small unit tests and do log analysis on the debug files afterwards.  This PR changes that to run a microbenchmark so we want to turn down debugging for that stage.  It's simply re-using the server_debug flag to say "I want less logging", a separate flag would be possible, but that would mean that we just needed to provide both of them together.